dashboard-frontend/src/helpers/state.spec.ts

```diff
@@ -77,7 +72,7 @@ describe('getFieldState', () => {
 
     const res = getFieldState(conf, localFormData);
 
-    const resValues = Object.values(res.map(f => f.value));
+    const resValues = Object.values(res.map((f: FieldState) => f.value));
```

Comment:
Not sure I understand what object.values does here, given that it's being passed an array?

Reply:
True it's not needed, I've removed it